src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java

```diff
@@ -71,4 +75,8 @@ public MixinEntityPlayer(World worldIn) {
         return this.openContainer == this.inventoryContainer;
     }
 
+    public String human$getName() {
+        return this.getName();
```

Comment:
Like https://github.com/SpongePowered/Sponge/commit/07d73f4b74f7851bf23b81bd01a4998a6d9df8fc#commitcomment-9648317, this will cause an infinite loop in the deobfuscated environment because `this.getName()` and `human$getName()` will be the same. You need to implement it like https://github.com/SpongePowered/Sponge/commit/07154247ec9fe21e81117a43a2326de0e20c3bf7